plugins/catalog-backend-module-msgraph/api-report.md

```diff
@@ -206,7 +220,7 @@ export type OrganizationTransformer = (
   organization: MicrosoftGraph.Organization,
 ) => Promise<GroupEntity | undefined>;
 
-// @public
+// @public @deprecated
```

Comment:
We could remove a lot from this API surface / api-report if we consider this as a breaking change without backwards compatibility.

Reply:
This module is not considered stable so breaking changes are "okay" but of course if we can give the user some heads up or at least a simple migration that would be preferable. Extra effort of deprecation vs hard breakage is always a tradeoff. I think we should remove the processor after some time of deprecation but the configuration itself 🤷